File: lib/Verify.js

function verify(yzm) {
    var v = session.get('__verify');
    if (v) {
        if (v === yzm.toUpperCase()) {
            //清空，防止多次使用
            session.set('__verify', null);
            return true;
        }
    }
    return false;
}

exports.verify = verify;

File: lib/Verify.go
package lib

import (
	"strings"
)

func Verify(yzm string, ctx *HandlerContext) bool {
	var v, ok = ctx.GetSessionVal("__verify")
	if ok {
		if v == strings.ToUpper(yzm) {
			//清空，防止多次使用
			ctx.SetSessionVal("__verify", nil)
			return true
		}
	}
	return false
}
